### ISSUE TITLE
Add New Game Confimation Dialog if there is a saved game.

### ISSUE THREAD

**Issue by lcenine** [TerryCavanagh/diceydungeons.com#1884]
This is a QoL addition not a  bug.

There have just been one too many times I've accidentally clicked New Game instead of Continue and there is no way to undo that.

An "Are You Sure?" confirmation would be great.

**Comment by ncrecc** [comment]
It used to be that the continue save wouldn't be erased until you actually started a new run as a new character, instead of once you clicked "new game". Perhaps that behavior should be back.

**Comment by TerryCavanagh** [comment]
Yeah, it probably shouldn't actually delete the save until you start a new run.
